cmd/main.go

```diff
@@ -556,6 +562,19 @@ func initEncrypter() error {
 	return nil
 }
 
+func initCompressor() error {
+	var err error
+	key := viper.GetString(compressionMethod)
+	if key != "" {
```

Comment:
I think before this, the compressor was not optional. This could cause `compressor` to stay `nil` and then during `CreateBackup` the code would panic.